script.js

```diff
@@ -0,0 +1,34 @@
+const header = document.querySelector('header');
+const title = document.getElementById('logo');
+const button = document.querySelector('.menu-bars');
+
+// const main = document.querySelector("main");
```

Comment:
- [ ] Please make sure your code is clean before pushing it to Github. Kindly remove the commented line 😉.